docs/api/internal.json

```diff
@@ -53,7 +53,7 @@
                         "type": "object",
                         "properties": {
                           "id": {
-                            "type": "string",
+                            "type": "integer",
```

Comment:
@aaronfc, I [suggested updating it to int](https://github.com/Automattic/WP-Job-Manager/pull/2469#discussion_r1238917056), but given this conversation (p1687532087992389-slack-C053397HN30), should we revert it to `string` and update it in the API to be parsed to `string`?

Sorry for the probably wrong suggestion here, @mikeyarce! 🤭

Reply:
No problem!  Done in c645824841fb08978a41b4f9c6984790b32b55fe